deployment/express/README.md

```diff
@@ -0,0 +1,46 @@
+# Spline UI Express Server
+
+Express server for a production deploying of Spline UI.
+
+
+## The very first setup
+
+- `npm install`
+
+## Start the express server 
+
+- `node server.js`    
+
+After that you can find the app running at the next address: http://localhost:7070. 
+
+
+## server.js Environment variables
+
+| Name        | Description           | Default value  |
+| ------------- |-------------| -----:|
+| `PORT`      | The port on which server will be running. | `7070` |
+| `SOURCES`      | The path to the root app directory.      |  `../../dist/spline-ui`  |
+| `CONSUMER_API` | Spline Consumer API URI.      |    `http://localhost:8080/consumer` |
+
+
+#### Example
+
+Serving the app at the custom port `9090`, from the custom directory `/path/to/the/splin-ui/` and consumer API running at the custom uri `http://my-spline-instance.com/consumer`:
+
+`set PORT=9090 && set CONSUMER_API=http://my-spline-instance.com/consumer&& set SOURCES=/path/to/the/splin-ui/&& node server.js`
```

Comment:
minor: missing space character in "consumer&&"

Also, is it a _Windows_ syntax? In unix `set` has a different semantics.

Reply:
One more thing about naming - especially when it comes to environment variable naming the more specific names would be preferred, to avoid potential collisions. I'd use "SPLINE_" prefix for example.

Reply:
@wajda 

1. No, there is no missing space, having no space is a correct syntax
2. Yes, this is windows syntax. But, I guess, it should work also for Linux if it is defined as a script cmd in package.json, but I am not sure. Anyway, this is just an example.
3. Agree, having a preffix is a good idea, will add it.